test/resources/flavors/real-test-flavor/real_flavor_base/language_definitions.json

```diff
@@ -8,6 +8,10 @@
       "parameters": [],
       "udf_client_path": {
         "executable": "/exaudf/exaudfclient"
+      },
+      "deprecation": {
+        "deprecation_date": "2024-10-31",
+        "default_changed_to": "Java 17"
```

Comment:
Hm, now is the question, what do we put here in practice. Should the SLC have a name field, too?

Reply:
This is more an informative string, not something we need to analyze in some code later. I believe not having any relation to any other field also gives us more freedom to customize this field however we want.

Reply:
Yeah, WE need anyway a strategy